hangman.py

```diff
@@ -60,9 +48,10 @@ def is_word_guessed(secret_word, letters_guessed):
     returns: boolean, True if all the letters of secret_word are in letters_guessed;
       False otherwise
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    if set(secret_word) == set(secret_word) & set(letters_guessed):  # & means intersection of these 2 sets
```

Comment:
As I remember it's not a best practice - commenting in the same line with code

Reply:
Made comment above line